### DISCUSSION TITLE
I can no longer create a new respository

### DISCUSSION THREAD

**Discussion by LauraHexx** [community/community#51112] [Repositories] [answered]
### Select Topic Area

Question

### Body

Hey
I can no longer create a new respository. Here is what I have done: After I created a new one, I deleted it again directly. A few minutes later I wanted to create a new one (with the same name - I think, but I'm not sure anymore). But that did not work. Now I only get the attached error message
![Bild1](https://user-images.githubusercontent.com/109242784/227948628-26534e98-93c7-4717-b561-70ca6e79cdce.png)

Thank you 😊

**Comment by GyulyVGC** [comment, marked as answer]
It's a problem on GitHub side: see [here](https://www.githubstatus.com) @LauraHexx 
We just have to wait and hopefully it'll be solved soon.
Don't forget to mark this comment as answer so that other people can view this discussion as solved! 

![Screenshot 2023-03-27 at 15 05 15](https://user-images.githubusercontent.com/100347457/227947004-184645b4-1590-40d8-afe0-825a878b6934.png)

**Comment by LauraHexx** [comment]
Thank you!
